pkg/dashboard/functiontemplates/gitfunctiontemplatefetcher.go

```diff
@@ -70,6 +78,21 @@ func (gftf *GitFunctionTemplateFetcher) Fetch() ([]*FunctionTemplate, error) {
 }
 
 func (gftf *GitFunctionTemplateFetcher) getRootTree() (*object.Tree, error) {
+	if gftf.caCert != "" {
+		certPool := x509.NewCertPool()
+		ok := certPool.AppendCertsFromPEM([]byte(gftf.caCert))
+		if !ok {
+			panic("Failed to parse ca certificate")
```

Comment:
Please use go idiomatic way of returning error. you can use nuclio error pkg to do so by simply `return nil, errors.New("Failed to parse certificate authority")`